### PR TITLE
KIALI-834 Remove contextual info such as credentials from error messages

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

File: src/components/NamespaceFilter/NamespaceFilter.tsx [hunk 1]
```diff
@@ -68,8 +68,9 @@ export class NamespaceFilter extends React.Component<NamespaceFilterProps, Names
         this.setState({ filterTypeList: initialFilters });
       })
       .catch(error => {
-        console.error(error);
-        this.props.onError('Error fetching namespace list.');
+        const errMsg = API.getErrorMsg('Error fetching namespace list.', error);
```
Comment (lucasponce) [inline src/components/NamespaceFilter/NamespaceFilter.tsx]: I imagine this one doesn't come from axios, isn't it ?

Comment (jotak) [reply src/components/NamespaceFilter/NamespaceFilter.tsx]: Yes it does, as it's originated from `API.getNamespaces` so an axios call

**Comment by xeviknal** [comment]
@jotak I guess that using `API.getErrorMsg` you are preventing to show the whole response data, right?

**Comment by jotak** [comment]
@xeviknal yes exactly, the whole "error" contains actually a lot of contextual info including request headers (so, credentials/token), whereas with `API.getErrorMsg` it extracts just the error message.

**Pull request merged**